modules/hardware/nvidia-jetson-orin/jetson-orin.nix

```diff
@@ -14,14 +16,61 @@
       passthrough-patch = ./pci-passthrough-agx-test.patch;
       vfio-pci = "vfio-pci.ids=10ec:c82f";
       deviceTree = "tegra234-p3701-host-passthrough.dtb";
+      buspath = [
+        {
+          bus = "pci";
+          path = "0001:01:00.0";
+        }
+      ];
+      kernelParams = [];
+      wireless = {
+        enable = true;
+        networks."SSID_OF_NETWORK".psk = "WPA_PASSWORD";
+      };
     };
     "nx" = {
       flashArgs = ["-r" config.hardware.nvidia-jetpack.flashScriptOverrides.targetBoard "nvme0n1p1"];
       passthrough-patch = ./pci-passthrough-nx-test.patch;
+      # Multiple device passing option
+      #      vfio-pci = "vfio-pci.ids=10de:229c,10ec:8168";
       vfio-pci = "vfio-pci.ids=10ec:8168";
       deviceTree = "tegra234-p3767-host-passthrough.dtb";
+      buspath = [
+        # Multiple devices and path could be passed through this option
+        #        {
+        #          bus = "pci";
+        #          path = "0008:00:00.0";
+        #        }
+        {
+          bus = "pci";
+          path = "0008:01:00.0";
+        }
+      ];
+      kernelParams = [
+        "pci=nomsi"
+        "pcie_acs_override=downstream,multifunction"
+      ];
+      wireless = {
+        enable = false;
+      };
     };
   };
+  netvmExtraModules = [
+    {
+      # This part should be conditional for AGX 01:01 for NX 08:01
+      microvm.devices = somDefinition."${cfg.somType}".buspath;
+
+      # nixpkgs.config.allowUnfree = true;
```

Comment:
Is there a reason to `allowUnfree = false`? If yes, please remove commented out.